.github/workflows/build-images.yml

```
@@ -0,0 +1,268 @@
+          cache-to: type=registry,ref=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:test-runner-buildcache,mode=max
+          platforms: linux/amd64
+
+  summary:
+    name: Build Summary
+    runs-on: ubuntu-latest
+    needs: [build-base-browser, build-fastapi, build-browser-service, build-test-runner]
+    if: always()
+
+    steps:
+      - name: Generate summary
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:10:47: Double quote to prevent globbing and word splitting

(shellcheck)